services/ark-api/ark-api/src/ark_api/api/v1/queries.py

```diff
@@ -116,12 +122,19 @@ async def create_query(
         if query.evaluatorSelector:
             spec["evaluatorSelector"] = query.evaluatorSelector.model_dump()
         
+        # Handle streaming annotation
+        metadata = {
+            "name": query.name,
+            "namespace": namespace
+        }
+        if query.streaming:
```

Comment:
We have consts for annotations. also be clear - if the query is being created with streaming requested set the annotations